base/models.py

```diff
@@ -604,6 +604,39 @@ def save(self, **kwargs):
         super().save(**kwargs)
 
 
+class Cancel(models.Model):
```

Comment:
```suggestion
class PlayerCancelLesson(models.Model):
```
название `Cancel` слишком общее и ни о чем не говорит. Нужно стараться писать код так, чтоб все из названий было понятно.